postgis/postgis.sql.in

```diff
@@ -2859,6 +2859,24 @@ CREATE OR REPLACE FUNCTION ST_Transform(geom geometry, from_proj text, to_srid i
 	LANGUAGE 'sql' IMMUTABLE STRICT PARALLEL SAFE
 	_COST_HIGH;
 
+CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys(auth_name text)
+	RETURNS SETOF TEXT
+	AS 'MODULE_PATHNAME', 'postgis_srs_codes'
```

Comment:
I would not use the same function name for a function that has a different return type.
Also, would it make sense to include "proj" in the function name, to be clear about what is this all about ?
`postgis_proj_<something>`